test/blackbox-tests/test-cases/package-rule.t/run.t

```diff
@@ -0,0 +1,5 @@
+When --only-packages is passed, it runs
+
+  $ dune build --only-packages a @runtest
+  A
+  A
```

Comment:
Can you make sure that this test actually succeeds in the commit where you are adding it. Otherwise it is difficult to see how the change changes things.

Reply:
Ah, I see your point. Made the test succeed with the failing state first before fixing it in the subsequent commit.